includes/fonction.php

```diff
@@ -62,7 +65,8 @@ function header_error($additional_head = '')
 
     $type_bottom = 'error';
 
-    $title = 'Libertempo';
+    $config = new \App\Libraries\Configuration(\includes\SQL::singleton());
+    $title = $config->getNomAppli();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused local variables such as '$title'.](https://app.codacy.com/manual/Libertempo/web/pullRequest?prid=4965647)

Reply:
Thank you @prytoegrian.
Thus I'm not sure how to correct this warning, it certainly existed before my commit ?

Reply:
Hi @benrubson
Thanks you for this PR. 
Don't pay attention to the $title, it's not pryto but a bot.

Reply:
OK great, thank you @wouldsmina 👍